dynawo/sources/API/CSTR/CSTRJsonExporter.cpp

```diff
@@ -0,0 +1,108 @@
+//
+// Copyright (c) 2015-2025, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+// This file is part of Dynawo, an hybrid C++/Modelica open source time domain
```

Comment:
// This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools
// for power systems.